insolar/assured-ledger
# PLAT-362: Change proto generation for messages via protoc-gen-ins

**- What I did**
Change proto generation for messages via protoc-gen-ins

**- How I did it**
- Change command in make file
- Change payload.proto and payload.r0.proto 
   - use insproto instead gogoproto
   - add option (insproto.id) instead Polymorph
   - add casttype for simple fields
   - add alias for message registration
- Remove PolymorphId

**- How to verify it**
run: make generate-protobuf (you should not see any changes in generated files)
run: make test_all

**- Description for the changelog**

See more task details:
https://insolar.atlassian.net/browse/PLAT-362
